docs/io/votable/table_element.rst

```diff
@@ -0,0 +1,245 @@
+.. doctest-skip-all
+
+Table element
+-------------
+
+VOTable files can contain ``RESOURCE`` elements, each of
+which may contain one or more ``TABLE`` elements. The ``TABLE``
+elements contain the arrays of data.
+
+To get at the ``TABLE`` elements, you can write a loop over the
+resources in the ``VOTABLE`` file::
+
+    for resource in votable.resources:
+        for table in resource.tables:
+            # ... do something with the table ...
+            pass
+
+However, if the nested structure of the resources is not important,
+you can use `~astropy.io.votable.tree.VOTableFile.iter_tables` to
+return a flat list of all tables::
+
+    for table in votable.iter_tables():
+        # ... do something with the table ...
+        pass
+
+Finally, if you expect only one table in the file, it might be most convenient
+to use `~astropy.io.votable.tree.VOTableFile.get_first_table`::
+
+  table = votable.get_first_table()
+
+Alternatively, there is a convenience method to parse a VOTable file and
+return the first table all in one step::
+
+  from astropy.io.votable import parse_single_table
+  table = parse_single_table("votable.xml")
+
+From a `~astropy.io.votable.tree.TableElement` object, you can get the data itself
+in the ``array`` member variable::
+
+  data = table.array
+
+This data is a ``numpy`` record array.
+
+The columns get their names from both the ``ID`` and ``name``
+attributes of the ``FIELD`` elements in the ``VOTABLE`` file.
+
+..
+  EXAMPLE START
+  Reading a VOTable File with astropy.io.votable
```

Comment:
@pllim - are we getting rid of these or not yet?

Reply:
I am not the correct person to ask. I don't know what "EXAMPLE START" is supposed to do. I forgot who added it now and couldn't dig it out with basic PR search.